src/test/java/SumWithForkJoinTest.java

```diff
@@ -0,0 +1,20 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class SumWithForkJoinTest {
+    private static int expected;
+    private static SumWithForkJoin sumWithForkJoin;
+
+    @BeforeAll
+    static void beforeAll() {
+        sumWithForkJoin = new SumWithForkJoin(Util.generateElements());
+        expected = Util.generateElements().stream().reduce(Integer::sum).get();
+    }
+
+    @Test
+    void sum_Ok() {
```

Comment:
idally we should have more than 1 test case

Reply:
I know, but what else can I check here ?

Reply:
in general, during testing you should think about valid data, invalid data, different combinations of valid data (negative numbers, overflow, etc) but for now you can leave it as it is